.dprint.json

```diff
@@ -13,9 +13,6 @@
     "associations": "**/*.rs",
     "rustfmt": "rustfmt --config imports_granularity=item"
   },
-  "includes": [
-    "**/*.{ts,tsx,js,jsx,json,md,toml,rs}"
-  ],
```

Comment:
Removed the need for an "includes" pattern. It's now constructed based on the plugins provided.

This revealed that we weren't formatting mjs files.